### PR TITLE
Aws DynamoDb Driver

### PR DESCRIPTION
**Pull request by faabiosr**
- Ignore Vim files
- Update composer with Aws packages
- Driver implementation
- Custom style

### CODE REVIEW

**Comment by augustohp** [comment]
panda-lambendo

File: src/Driver/DynamoDb/Driver.php [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    /**
+     * @var \Aws\DynamoDb\Marshaler
+     */
+    private $marshaler;
+
+    /**
+     * Driver constructor.
+     *
+     * @param \Aws\DynamoDb\DynamoDbClient $connection
+     * @param string                       $database
```
Comment (tonicospinelli) [inline src/Driver/DynamoDb/Driver.php]: you forgot to remove this parameter

**Comment by tonicospinelli** [comment]
hey @fabiorphp great job!!! It is awesome!!!

Your changes has a little mistakes from code style, could you run php-cs-fix before I merge it?

``` sh
composer csfix
```

**Comment by augustohp** [comment]
> Your changes has a little mistakes from code style, could you run php-cs-fix before I merge it?

Can't we add this as a status check, preventing merges with bad CS?

**Comment by faabiosr** [comment]
Done!

**Comment by henriquemoody** [comment]
> Can't we add this as a status check, preventing merges with bad CS?

We can do better, [StyleCI](https://styleci.io/) can fix the standards then we don't need to bother contributors too much about coding standards.

**Comment by tonicospinelli** [comment]
@fabiorphp :clap: thanks!!

@henriquemoody @augustohp  I didn't know about this tool for code style, I'll read about and try to use it in this repo :smile:

**Pull request merged**